src/Components/Home.tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState, useRef } from 'react'
```

Comment:
Also As this home component is not a reusable component as Page in itself.

could you reposition this file to 

src/pages/

instead of 

src/components/